kmip/core/enums.py

```diff
@@ -330,10 +330,9 @@ class SecretDataType(Enum):
     PASSWORD = 0x00000001
     SEED     = 0x00000002
 
-
 # 9.1.3.2.10
 class OpaqueDataType(Enum):
-    pass
+    NONE = 0x00000000 # Not defined by the standard, but we need something
```

Comment:
I'm afraid if we define this to 0x00 then a later standard may override this value. Have you considered using 0x8XXXXXXX? This is the pattern for extensions.

I find it bizarre in the KMIP spec that the opaque data type, which is an object that may not be interpreted by the server, requires an opaque data type. That's my two cents on the spec.

Does the kmip test documentation have an example for opaque data?